src/templates/next/components/complex/Hero/Hero.tsx

```diff
@@ -23,7 +23,7 @@ const HeroGradient = ({
 }: HeroGradientProps) => {
   return (
     <section
-      className="bg-cover bg-center bg-no-repeat h-[32rem]"
+      className="bg-cover bg-center bg-no-repeat h-auto sm:h-[22.5rem] md:h-[31.25rem]"
```

Comment:
question: I think these should be max-height?
- Desktop: 500px
- Tablet: 360px
- Mobile: 240px

Reply:
the ticket (here)[https://linear.app/ogp/issue/ISOM-993/hero-banner-design-review] is min h leh